python/lsst/ip/isr/linearize.py

```diff
@@ -56,9 +62,14 @@ class Linearizer(abc.ABC):
         Raised if the supplied table is not 2D, or if the table has fewer
         columns than rows (indicating that the indices are swapped).
     """
+
+    _OBSTYPE = "linearizer"
+    """The calibration type used for ingest."""
```

Comment:
Might be more accurate to call this the "dataset type name used for this class".